grpc/oob/common.go

```diff
@@ -10,6 +10,11 @@ import (
 	"github.com/tinkerbell/pbnj/pkg/repository"
 )
 
+var (
+	// DefaultBMCTimeout is the default value for how long a BMC call/interaction is allowed to run before it is cancelled.
+	DefaultBMCTimeout = 120 * time.Second
```

Comment:
Hey @jacobweinstock I would have liked to do that, although, since I'm trying to make sure we don't have the default value defined separately, it made sense to have `DefaultBMCTimeout` exported

https://github.com/joelrebel/pbnj/blob/0e04e87687f2c2267a8569cc165ef957f5791c70/grpc/server.go#L73
https://github.com/joelrebel/pbnj/blob/0e04e87687f2c2267a8569cc165ef957f5791c70/cmd/server.go#L116
https://github.com/joelrebel/pbnj/blob/0e04e87687f2c2267a8569cc165ef957f5791c70/grpc/oob/common.go#L121

It could be an exported method that returns the timeout duration, unless you have other ideas...

Reply:
sure a function works, or even a const.

Reply:
thanks, fixed now.